lerna.json

```diff
@@ -0,0 +1,10 @@
+{
+  "packages": [
+    "apps/public-reference",
+    "services/listings",
+    "shared/*"
+  ],
+  "version": "0.0.9",
+  "npmClient": "yarn",
+  "useWrokspaces": true
```

Comment:
I think the `useWrokspaces` line should be removed…?

Reply:
I think you are right.

Reply:
My reading of the docs is that we probably do want to useWorkspaces (note: spelled correctly), especially since we're already using them -- unless there's something I'm missing?

Reply:
We can either use useWorkspaces flag or define packages. Both ways should work the same.